### ISSUE TITLE
[Feature Request] Create ARM64 build for other OS

### ISSUE THREAD

**Issue by xyx-bbl** [usebruno/bruno#335]
Right now, it seems that there are only ARM64 build for macOS. It would be helpful if you could build ARM64 binaries for other operating systems (like Linux) for this project as well, so that people working on ARM devices wouldn't have to compile this awesome project from source.

**Comment by helloanoop** [comment]
Sure. This week priority is to get the remaining critical features done.

Next week I will be focusing on setting up binaries across various package distributions

**Comment by jokaorgua** [comment]
are there any news here? will there be a build for ubuntu arm ? (aarch64)

**Comment by theZappr** [comment]
Awesome project :+1: 

My daily driver is now a Pi5 so a deb ARM64 of this would be sweet! If I knew how to do a debian/ubuntu arm build I would PR it but I don't...so I wont  :(

**Comment by captn3m0** [comment]
Would appreciate this as well, using Asahi Linux which needs a aarch64 build.

**Comment by brad** [comment]
I wanted this too and was having trouble getting it to work locally so I set up [this branch](https://github.com/brad/bruno/tree/feature/linux-arm64-build) with a POC circleci pipeline to make ARM64 linux builds (the github arm64 runner is in private beta still). You can find the artifacts [here](https://app.circleci.com/pipelines/github/brad/bruno/20/workflows/6c02dfcb-50b4-493e-a4d3-9e980e54d7d2/jobs/17/artifacts). Fair warning, I've only tested the deb

**Comment by richardfsr** [comment]
rpm build confirmed working too @brad

**Comment by luisnabais** [comment]
Is there any arm64 artifact for Debian, flatpak or AppImage? Any information when more arm64 Linux builds will officially be available? Thanks

**Comment by minzdrav** [comment]
Hi Bruno team
It would be nice to have native Windows arm64 builds.
I have one of those Snapdragon X Elite laptops and can run a test build if you want to check if it works before public release.

**Comment by gauravmahto** [comment]
Hi Bruno team,

Can we get a arm64 build for Windows. Let me know if I can help?

**Comment by Painkiller995** [comment]
+1 vote

**Comment by riotejas** [comment]
> Is there any arm64 artifact for Debian, flatpak or AppImage? Any information when more arm64 Linux builds will officially be available? Thanks

+1 for arm64 for linux.   I would like to run in ubuntu VM on apple silicon.

**Comment by Siilwyn** [comment]
Running [Asahi Linux](https://asahilinux.org/) an ARM flatpak build would be great!

**Comment by neoix** [comment]
I have a Surface Laptop with ARM CPU. Can we have an installer compatible with Windows ARM64 please? @helloanoop

**Comment by jensbech** [comment]
+1 for arm64 Linux.

**Comment by helloanoop** [comment]
We are focusing on this during our December sprint. @anusree-bruno will share updates here in the coming days.

**Comment by asoltesz** [comment]
+1 for the Linux arm64 version.

I for example work on a company supplied Macbook but since I cannot stand MacOS, I develop in a graphical Linux VM.
(so I couldn't even try it out)

**Comment by brad** [comment]
> I for example work on a company supplied Macbook but since I cannot stand MacOS, I develop in a graphical Linux VM.

@asoltesz I thought I was the only one! 🤗

**Comment by bluegod** [comment]
@anusree-bruno any updates on this? Thank you!

**Comment by anusree-bruno** [comment]
Hi @bluegod

@ramki-bruno from our team is currently handling this. I'll share an update in a few days, and we expect to have it completed by the end of this month.

**Comment by nikleonard** [comment]
> > I for example work on a company supplied Macbook but since I cannot stand MacOS, I develop in a graphical Linux VM.
> 
> [@asoltesz](https://github.com/asoltesz) I thought I was the only one! 🤗

No, you are not the only one. I ended compiling Bruno from source a while ago (1.34.1) and it works fine in Ubuntu ARM64 running in UTM on apple silicon. I'm just waiting for the official .deb package because I don't want to rebuild it again (and I forgot how I built it, but I remember having to solve some issues with the electron build tool chain).

**Comment by brad** [comment]
@nikleonard As I mentioned before in this thread I've created a [branch](https://github.com/brad/bruno/tree/feature/linux-arm64-build) where I'm able to build on linux so you're welcome to look at that to save some time. It also has a [circleci](https://app.circleci.com/pipelines/github/brad/bruno) integration and I just kicked off another build. I've been happily using the deb from circleci on Ubuntu ARM64 inside parallels for months

**Comment by ramki-bruno** [comment]
We will be having `aarch64`/`arm64` builds for `.AppImage`, `.deb` and `.rpm` in our next [nightly-build](https://github.com/usebruno/bruno-nightly-builds/releases) and if everything goes well, the same will also be included in out next stable release.

For Windows-Arm64, we need bit more time.
Also, will look into Snapcraft and Flatpak builds as I can see we are missing it for last couple of releases.

**Comment by ramki-bruno** [comment]
> We will be having aarch64/arm64 builds for .AppImage, .deb and .rpm in our next [nightly-build](https://github.com/usebruno/bruno-nightly-builds)

This is already out, please try it out. [Download links are here.](https://github.com/usebruno/bruno-nightly-builds)

**Comment by PsyChonek** [comment]
Hi, is it possible to also add Windows ARM64? Thank you.

**Comment by allensmith81** [comment]
Any word on a Windows Arm64 Built?

**Comment by rs38** [comment]
for me it took like ~1h of vibe buiding in vscode agent mode to get it somehow build to Winarm Electron.

<img width="395" height="81" alt="Image" src="https://github.com/user-attachments/assets/64e6e6b8-25bf-4f8a-a77d-1b12064bdb58" />

issues at least:
- Version info does not reflect the lastest release from main 

draft: https://github.com/usebruno/bruno/pull/5297

**Comment by allensmith81** [comment]
Exciting now all we need is an official build :)

**Comment by rs38** [comment]
https://github.blog/changelog/2025-04-14-windows-arm64-hosted-runners-now-available-in-public-preview/

the needed environment is quite "fresh", and the amount of consumers not too high. 
I am pretty much convincend of the winarm speed for a 700 EUR notebook though, outperforms a 4 year old i7 Surface Book easily.

**Comment by allensmith81** [comment]
Let me know if you want anyone to test :P

**Comment by rs38** [comment]
way easier workaround with WSL2 and Ubuntu distri installed as there is X server support for 1-2 years:

download latest debian arm64 binary and install it:
` sudo apt install ./bruno_2.10.0_arm64_linux.deb`

then you can run arm64 Bruno from whereever you like (create a shortcut on you windows desktop with
 `wsl -e bruno --disable-gpu`



<img width="1395" height="985" alt="Image" src="https://github.com/user-attachments/assets/2d1c3b51-971d-448b-bf63-3103d1a59d49" />

@ramki-bruno may be worth adding this to some docs?

**Comment by philippebn** [comment]
Very interest about a Windows Arm64 Built

**Comment by bpacholek** [comment]
@philippebn @warren-xyx Electron for WIndows supports now ARM64 so it has been already added as a standalone issue:
https://github.com/usebruno/bruno/issues/6302
and merged:
https://github.com/usebruno/bruno/pull/6303

Already possible to build from current source, integrated into Bruno's build process so next release should provide official ARM64 release.

**Comment by Pragadesh44-Bruno** [comment]
The Windows builds for ARM64 are available starting with today’s nightly builds. https://github.com/usebruno/bruno-nightly-builds/releases/tag/v2.15.1-2025.12.12. Thanks for the contribution @bpacholek

**Comment by xsa91** [comment]
Yet not available for Linux-ARM64.

**Comment by rs38** [comment]
it is, see above: https://github.com/usebruno/bruno/issues/335#issuecomment-3239913415
